### PR TITLE
Add more checks to pre-commit

### PR DESCRIPTION
**Pull request by alanssitis**
## Short description
Added xml checks and prettier.

## Task checklist
- [x] Fix weird formatting of `*.inc` files.

### CODE REVIEW

File: src/sim_stage/sim_files/carlike_robot.inc [hunk 1]
```diff
@@ -1,38 +1,10 @@
...
+define laser ranger ( sensor ( range_max 6.5 fov 180 samples 640 ) # generic
+model properties color "black" size [ 0.06 0.15 0.03 ] ) # # Robot model: #
+footprint (counter-clockwise): [-0.1,-0.125], [0.5,-0.125], [0.5,0.125],
+[-0.1,0.125] # center of rotation: [0,0] # wheelbase: 0.4 define carlike_robot
+position ( pose [ 0.0 0.0 0.0 0.0 ] odom_error [0.03 0.03 999999 999999 999999
+0.02] size [ 0.6 0.25 0.40 ] # this models the footprint (rectangular), but
+shifted such that the bottom-left corner is in [0,0]. The center of rotation now
+here at [0.3, 0.125, 0.2] # correct center of rotation: origin [ 0.2 0.0 0.0
+0.0] gui_nose 1 color "red" # kinematics drive "car" wheelbase 0.4 # distance
+between both axles # spawn sensors laser(pose [ -0.1 0.0 -0.11 0.0 ]) )
```
Comment (alanssitis) [inline src/sim_stage/sim_files/carlike_robot.inc:10]: What does this file do @Reschivon ? Am having a hard time finding a reference to it.

File: src/sim_stage/sim_files/map.inc [hunk 2]
```diff
@@ -1,14 +1,3 @@
...
-	gui_grid 0
-	gui_move 0
-	gui_outline 0
-	gripper_return 0
-	fiducial_return 0
-	laser_return 1
-)
+define floorplan model ( # sombre, sensible, artistic color "gray30" # most maps
+will need a bounding box boundary 1 gui_nose 0 gui_grid 0 gui_move 0 gui_outline
+0 gripper_return 0 fiducial_return 0 laser_return 1 )
```
Comment (alanssitis) [inline src/sim_stage/sim_files/map.inc:3]: ditto

**Review by alanssitis — COMMENTED** [review]
@Reschivon these should be deleted if unecessary.

**Comment by alanssitis** [comment]
> @Reschivon these should be deleted if unecessary.

Looks like it's some weird syntax http://wiki.ros.org/stage_ros#World_syntax. Should be removed in the future when we move to a better sim and delete this.

**Review by xclucas — APPROVED** [review]
I am pretty tired but I looked through it and it's all just styling

**Comment by Reschivon** [comment]
> @Reschivon these should be deleted if unecessary.

Ah yeah the stage simulator. Takes me back. It had its time but we will probably never use it again

**Pull request merged**